assets/src/edit-story/components/panels/design/layerStyle/layerStyle.js

```diff
@@ -35,36 +36,35 @@ const MIN_MAX = {
   },
 };
 
+const Wrapper = styled.div`
+  width: 128px;
+`;
+
 function defaultOpacity({ opacity }) {
   return typeof opacity !== 'undefined' ? opacity : MIN_MAX.OPACITY.MAX;
 }
 
 function LayerStylePanel({ selectedElements, pushUpdate }) {
   const opacity = getCommonValue(selectedElements, defaultOpacity);
 
-  usePresubmitHandler(({ opacity: newOpacity }, { opacity: oldOpacity }) => {
-    const value =
-      typeof newOpacity === 'undefined' || typeof oldOpacity === 'undefined'
-        ? MIN_MAX.OPACITY.MAX
-        : newOpacity;
-    return {
-      opacity: clamp(value, MIN_MAX.OPACITY),
-    };
-  }, []);
+  const handleChange = useCallback(
+    (evt, value) => pushUpdate({ opacity: value ?? 100 }),
```

Comment:
Looks like setting the opacity is not working well now -- even when tabbing out of the input, the opacity of the element doesn't change:

![Mar-03-2021 13-09-56](https://user-images.githubusercontent.com/3294597/109797196-6c6c5e00-7c19-11eb-841f-87b36196a615.gif)

Reply:
Ah thanks, fixed it in d9975e8fe.